src/app/docs/kagent/supported-providers/gemini/page.mdx

```diff
@@ -28,19 +28,28 @@ Make sure that your Google Cloud account has a project with the Gemini API enabl
 
 3. Create a ModelConfig resource using the `Gemini` provider. The following Gemini models are supported:
 
-   - **gemini-pro** - Text generation with function calling support
-   - **gemini-pro-vision** - Multimodal model for text and image processing
-
-   ```yaml
-   apiVersion: kagent.dev/v1alpha2
-   kind: ModelConfig
-   metadata:
-     name: gemini-model-config
-     namespace: kagent
-   spec:
-     apiKeySecret: kagent-gemini
-     apiKeySecretKey: GOOGLE_API_KEY
-     model: gemini-pro
-     provider: Gemini
-     gemini: {}
-   ```
+All gemini models support multimodal inputs (imags, audio, video, text), more information regarding model choices can be found on the [Gemini API docs](https://ai.google.dev/gemini-api/docs/models).
+
+- **gemini-2.5-pro** - Thinking and reasoning, multimodal understanding, advanced coding
```

Comment:
you can probably remove the list here now - that way, users can click on the link and always get the latest model name :)